### PR TITLE
enhance: Support importing data with parquet file

### PR DESCRIPTION
**Pull request by xiaocai2333**
issue: #28272

Numpy does not support array type import. 
Array type data is imported through parquet.

### CODE REVIEW

File: internal/util/importutil/import_util.go [hunk 1]
```diff
@@ -112,6 +113,10 @@ func initBlockData(collectionSchema *schemapb.CollectionSchema) BlockData {
 			blockData[schema.GetFieldID()] = &storage.JSONFieldData{
 				Data: make([][]byte, 0),
 			}
+		case schemapb.DataType_Array:
+			blockData[schema.GetFieldID()] = &storage.ArrayFieldData{
+				Data: make([]*schemapb.ScalarField, 0),
```
Comment (czs007) [inline internal/util/importutil/import_util.go:118]: not initialize ElementType?

**Comment by xiaocai2333** [comment]
rerun ut

**Comment by xiaocai2333** [comment]
rerun ut

**Comment by xiaocai2333** [comment]
rerun ut

File: internal/util/importutil/parquet_parser.go [hunk 2]
```diff
@@ -0,0 +1,932 @@
...
+			segmentData[reader.fieldID] = fieldData
+		}
+
+		// nothing to read
+		if readRowCount == 0 {
+			break
+		}
+		totalRead += readRowCount
+		updateProgress(int64(totalRead))
+		tr.Record("readData")
```
Comment (czs007) [inline internal/util/importutil/parquet_parser.go:310]: usesless log?

**Pull request merged**